### ISSUE TITLE
Why the parent of deformed bones is not the root bone?

### ISSUE THREAD

**Issue by iillya** [Arminando/GameRig#22]
<img width="1935" alt="Snipaste_2023-03-02_11-02-47" src="https://user-images.githubusercontent.com/112866800/222320955-482b2ad3-03c3-48ae-b382-a949f92ae2d7.png">
Why the parent of deformed bones is not the root bone?
UE engine needs the root bone animes.

**Comment by arminHalac** [comment]
The "root" bone is just a control bone which will be deleted on export. The "DEF-char_root" bone is the actual root that is supposed to be used in UE.

**Comment by iillya** [comment]
> The "root" bone is just a control bone which will be deleted on export. The "DEF-char_root" bone is the actual root that is supposed to be used in UE.

But the default root bone is more intuitive as the root bone than adding an extra one,and it conforms to most naming conventions

**Comment by arminHalac** [comment]
I agree. It is not ideal. The only reason it is like this is that rigify hard coded the name "root" into the world control and I did not have time to restructure the whole thing.

**Comment by iillya** [comment]
> I agree. It is not ideal. The only reason it is like this is that rigify hard coded the name "root" into the world control and I did not have time to restructure the whole thing.

Hi,I have an idea.Is this okay？
https://youtu.be/kRjSQZbwHfw

**Comment by iillya** [comment]
> It seems to work fine. And whatever works is okay :)

Hi,I wrote a piece of code to reparent the bones.Are you considering adding this feature in the future?
<img width="1920" alt="Snipaste_2023-03-07_18-14-53" src="https://user-images.githubusercontent.com/112866800/223392513-06516580-0d69-474e-93a6-c2f128fa5bd9.png">

**Comment by neilfranci** [comment]
@iillya Do you use Game Rig Tool? It will generate a deformed game rig to use in the game engine. The one you're tweaking is just a control rig. I use Export to Godot and it works fine.
![blender_cDvYvfKYhb](https://github.com/Arminando/GameRig/assets/66520542/5d69ecbb-68c6-48de-9a8d-714f02c21148)
